easybuild/easyblocks/c/clang.py

```diff
@@ -215,21 +222,27 @@ def configure_step(self):
         self.cfg.update('configopts', "-DGCC_INSTALL_PREFIX='%s'" % gcc_prefix)
         self.log.debug("Using %s as GCC_INSTALL_PREFIX", gcc_prefix)
 
+        # Configure some default options
         self.cfg.update('configopts', "-DCMAKE_BUILD_TYPE=Release")
+        self.cfg.update('configopts', '-DLLVM_REQUIRES_RTTI=ON')
+        self.cfg.update('configopts', '-DLLVM_ENABLE_RTTI=ON')
+        self.cfg.update('configopts', '-DLLVM_ENABLE_EH=ON')
+        self.cfg.update('configopts', '-DCLANG_DEFAULT_OPENMP_RUNTIME=libomp')
+
         if self.cfg['assertions']:
             self.cfg.update('configopts', "-DLLVM_ENABLE_ASSERTIONS=ON")
         else:
             self.cfg.update('configopts', "-DLLVM_ENABLE_ASSERTIONS=OFF")
 
-        if self.cfg["build_lld"]:
-            self.cfg.update('configopts', "-DLLVM_ENABLE_PROJECTS=lld")
```

Comment:
Why is this being dropped? Is it not required when `build_lld` is enabled after all?

Reply:
I verified that I got both libc++ and lld when I used this for my Clang PR.

Reply:
It's because I placed the lld sources in a different directory, when they are placed there there is no longer a need for this option.

Reply:
ok, thanks for clarifying